src/landscape/components/LandscapeView.js

```diff
@@ -68,16 +69,24 @@ const LandscapeCard = ({ landscape }) => {
   );
 };
 
-const LandscapeMap = ({ position }) => {
+const LandscapeMap = ({ landscape }) => {
   const { t } = useTranslation();
-  const bounds = position && [
-    [position.boundingbox[0], position.boundingbox[2]],
-    [position.boundingbox[1], position.boundingbox[3]],
+  const { areaPolygon, position } = landscape;
```

Comment:
suggestion: I think this logic to calculate the bounds can be extracted to a separate file, maybe to `src/landscape/polygonUtils.js` with this also unit tests could be added to validate how the areaPolyong or poistion is transform to the bounds value